applications/rag/tests/test_rag.py

```diff
@@ -3,160 +3,125 @@
 import requests
 
 def test_prompts(prompt_url):
```

Comment:
For the tests, can we do similar test as before, i.e. asserting the response with an pre-defined expected response?

Reply:
@gongmax I don't think it's a good approach due to the probabilistic nature of the LLM, it could give different answers affecting the stability of the test.